composer.json

```diff
@@ -23,33 +23,35 @@
     "config": {
         "sort-packages": true
     },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "3.3-dev",
-            "dev-develop": "4.0-dev"
-        }
-    },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.3 || ~8.0.0",
         "container-interop/container-interop": "^1.2",
         "laminas/laminas-stdlib": "^3.2.1",
         "laminas/laminas-zendframework-bridge": "^1.0",
         "psr/container": "^1.0"
     },
     "require-dev": {
+        "composer/package-versions-deprecated": "^1.0",
         "laminas/laminas-coding-standard": "~1.0.0",
-        "mikey179/vfsstream": "^1.6.5",
-        "ocramius/proxy-manager": "^1.0 || ^2.0",
-        "phpbench/phpbench": "^0.13.0",
-        "phpunit/phpunit": "^5.7.25 || ^6.4.4"
+        "laminas/laminas-container-config-test": "^0.3",
+        "laminas/laminas-dependency-plugin": "^2.1",
+        "mikey179/vfsstream": "^1.6.8",
+        "ocramius/proxy-manager": "^2.2.3",
+        "phpbench/phpbench": "^1.0.0-alpha3",
+        "phpspec/prophecy-phpunit": "^2.0",
+        "phpunit/phpunit": "^9.4"
     },
     "provide": {
         "container-interop/container-interop-implementation": "^1.2",
         "psr/container-implementation": "^1.0"
     },
+    "conflict": {
+        "zendframework/zend-code": "<3.3.1",
+        "laminas/laminas-code": "<3.3.1"
+    },
     "suggest": {
-        "laminas/laminas-stdlib": "laminas-stdlib ^2.5 if you wish to use the MergeReplaceKey or MergeRemoveKey features in Config instances",
-        "ocramius/proxy-manager": "ProxyManager 1.* to handle lazy initialization of services"
+        "laminas/laminas-stdlib": "laminas-stdlib ^2.7.7 | ^3.1 if you wish to use the MergeReplaceKey or MergeRemoveKey features in Config instances",
```

Comment:
The "require" section already contains laminas-stdlib therefore this line can be removed.